turbopack/crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -3201,7 +3199,7 @@ impl ModulePart {
     }
 
     pub fn internal_evaluation(id: u32) -> Self {
-        ModulePart::InternalEvaluation(id)
+        ModulePart::Internal(id)
     }
```

Comment:
We can remove that function entirely then if it's the same as the one above?

Reply:
Oops. Rebase mistake. I removed it.